cmd/manager/main.go

```diff
@@ -152,22 +150,11 @@ func main() {
 			log.Error(errors.New("timed out waiting for caches to sync"), "")
 			os.Exit(1)
 		}
-		// create the TektonInstallation resource on the cluster at startup, stop if something when wrong
-		log.Info("Creating the Tekton installation resource")
-		if err = mgr.GetClient().Create(context.TODO(), tektoninstallation.NewInstallation()); err != nil && !apierrors.IsAlreadyExists(err) {
-			log.Error(err, "Failed to create the 'TektonInstallation' custom resource during startup")
-			os.Exit(1)
-		}
-		log.Info("Tekton Installation resource created")
 
-		// create the CheInstallation resource on the cluster at startup, stop if something when wrong
-		log.Info("Creating the Che installation resource")
-		if err = mgr.GetClient().Create(context.TODO(), cheinstallation.NewInstallation()); err != nil && !apierrors.IsAlreadyExists(err) {
-			log.Error(err, "Failed to create the 'CheInstallation' custom resource during startup")
+		if err := pkg.CreateInstallationResources(mgr.GetClient(), mgr.GetScheme(), log); err != nil {
+			log.Error(err, "unable to create toolchain installation resources")
```

Comment:
```suggestion
			log.Error(err, "unable to create toolchain installation resources during startup")
```

Reply:
(see other suggested changes below, in which the "during startup" words were removed)